bindings/jupyroot/python/JupyROOT/kernel/utils.py

```diff
@@ -65,7 +65,9 @@ def __init__(self,kernel):
          magics_path = os.path.dirname(__file__)+os.sep+"magics"+os.sep+"*.py"
          for file in glob(magics_path):
               if file != magics_path.replace("*.py","__init__.py"):
-                  module_path="JupyROOT.kernel.magics."+file.split(os.sep)[-1].replace(".py","")
+module_prefix = "JupyROOT.kernel.magics."
+module_name = os.path.splitext(os.path.basename(file))[0]
+module_path = module_prefix + module_name
```

Comment:
> I'm sorry I think in my previous comment the indentation got messed up, these lines should be properly indented under the `if` otherwise they won't work

Yes, I've seen that. This will be fixed in my next commit